docs/usage/configuration-options.md

```diff
@@ -1169,7 +1169,7 @@ For instance if you have a project with an `"examples/"` directory you wish to i
 }
 ```
 
-Useful to know: Renovate's default ignore is `node_modules` and `bower_components` only, however if you are extending the popular `config:base` preset then it adds ignore patterns for `vendor`, `examples`, `test(s)` and `fixtures` directories too.
+Useful to know: Renovate's default ignore is `node_modules` and `bower_components` only, but if you are extending the popular `config:base` preset then it adds ignore patterns for `vendor`, `examples`, `test(s)` and `fixtures` directories too.
```

Comment:
```suggestion
Renovate's default ignore is `node_modules` and `bower_components` only.
If you are extending from the popular `config:base` preset then it adds ignore patterns for `vendor`, `examples`, `test(s)` and `fixtures` directories too.
```

I'm rewriting this so we don't need `but/however`. 😏 
I'm also dropping `userful to know:` as we shouldn't be mentioning things that are not necessary to know. 😉